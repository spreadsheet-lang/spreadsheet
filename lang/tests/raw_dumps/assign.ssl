A1 = 0
A2 = 1
