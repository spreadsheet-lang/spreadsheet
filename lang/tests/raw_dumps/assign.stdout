ROOT@0..5
  STATEMENT@0..5
    ASSIGN@0..4
      CELL@0..2 "A1"
      EQ@2..3 "="
      INT@3..4 "0"
    NEWLINE@4..5 "\n"
